### ISSUE TITLE
`mobie.add_image` transformations

### ISSUE THREAD

**Issue by martinschorb** [mobie/mobie-utils-python#96]
Hi,

I tried adding some transformations to imported sources. It seems that the parameter in ` add_image` points to a XML-specific function.
https://github.com/mobie/mobie-utils-python/blob/977120d6b90154211e6c1c121a77e1d9e13e4186/mobie/image_data.py#L250

What is the best way to include transformation upon data import (multi-modal/ multi-scale registration)? Loading the view and modifying it afterwards?

**Comment by constantinpape** [comment]
> I tried adding some transformations to imported sources. It seems that the parameter in ` add_image` points to a XML-specific function.

Yes, indeed this currently only works for bdv.xml formats. I have started to generalize the functionality to also support ome.zarr here: https://github.com/mobie/mobie-utils-python/pull/94.
However, I realized that it currently doesn't make sense to enable this for ome.zarr, since the transformations supported by it are limited; and the v0.5 release which will significantly extend the transformation functionality in ome.zarr is due soon.
(I still went ahead and merged this, so that an error message is thrown when trying to add an ome.zarr image with transformation).

For some more discussions about the transformation functionality, see also https://forum.image.sc/t/mobie-utils-python-understanding-transformations-and-views/78341/4.

(And we can leave this issue open; I will try to implement the trafo support for ome.zarr as soon as v0.5 is released)
